app.js

```diff
@@ -1,38 +1,73 @@
 'use strict';
 
-let userName = prompt("What is your name?");
-document.write ("Hello " + userName + ", welcome to my page!");
-
-let farm = prompt("Do I want a farm?")?.toLowerCase();
-if (farm === "yes" || farm === "y") {
-    alert("Correct!");
-} else if (farm === "no" || farm === "n") {
-    alert("FALSE!! I said it like 3 times...");
-} else {
-  alert("(Y)es or (N)o");
+function Questions(answer) {
```

Comment:
Good use of a function! I like that you make one function and use it for all of the questions! However this function does limit you to only true answers, and doesn't keep track of score.